stdlib/builtins.pyi

```diff
@@ -73,7 +74,7 @@ _T4 = TypeVar("_T4")
 _T5 = TypeVar("_T5")
 _TT = TypeVar("_TT", bound="type")
 _TBE = TypeVar("_TBE", bound="BaseException")
-_R = TypeVar("_R")  # Return-type TypeVar
+_FuncT = TypeVar("_FuncT", bound=Callable[..., Any])
```

Comment:
Where is this used?

Reply:
Good catch, it's a left-over from before the merge.